robot/client/client.go

```diff
@@ -1190,6 +1192,64 @@ func (rc *RobotClient) Version(ctx context.Context) (robot.VersionResponse, erro
 	return mVersion, nil
 }
 
+// Tunnel tunnels data to/from the read writer from/to the destination port on the server. This
+// function will close the connection passed in as part of cleanup.
+func (rc *RobotClient) Tunnel(ctx context.Context, conn net.Conn, dest int) error {
```

Comment:
Is there any restriction on multiple tunnels? Can I open multiple tunnels to the same destination port at once?

Reply:
no restrictions - during testing, I find that browsers will open a lot of connections to the same port at the same time.